### ISSUE TITLE
Pending Rollback Error (random)

### ISSUE THREAD

**Issue by harmeet-status** [sartography/spiff-arena#778]
Got a Pending rollback error two times.
1. Executed cypress automated tests and in parallel created some PP1 requests manually. (on test.app) - happened on 14th November
2. When clicking on the Start New + button. No parallel activities are involved. (on dev.app) - happened on 8th November

**Comment by harmeet-status** [comment]
Comment from Dinithi:

Screenshots for Scenario 1

![Image](https://github.com/sartography/spiff-arena/assets/104047049/bc3e8f5b-d2a0-4de9-ba76-b4cf6c8e87fe)

![Image](https://github.com/sartography/spiff-arena/assets/104047049/9c74831e-2d2e-4649-a0d8-6f81dc4979c3)

**Comment by harmeet-status** [comment]
Comment from Dinithi:

Screenshot for Scenario 2 - on 8th November

![Image](https://github.com/sartography/spiff-arena/assets/104047049/cba0cfd7-afd3-46d0-9cc9-df1dc25746f4)

**Comment by danfunk** [comment]
Estimating only the task deadlock, will create a new ticket for the guid.

**Comment by jasquat** [comment]
Potentially fixed by #823
